server.js

```diff
@@ -1,6 +1,5 @@
 'use strict';
 
-const knex = require('./database');
 const express = require('express');
 const app = express();
 const bodyParser = require('body-parser');
```

Comment:
Instead of making urlParser a middleware for every route, you can use `app.use(urlParser)` before all your calls to your routes to enforce urlParser as a middleware for all lines of code below it.